lib/initializers/metrics.js

```diff
@@ -0,0 +1,17 @@
+var Initializer = require("../initializer");
+
+
+Initializer.add('startup', 'stex.metrics', ['stex.config', 'stex.logging'], function(stex) {
+  var statsd = stex.conf.get("statsd") || false;
+
+  if (statsd) {
```

Comment:
simply not creating a metrics instance in the case we don't have one configured isn't a good plan.  We should have a no-op sink instead, such that our intstumented code doesn't always have to do:

```javascript
if (stex.metrics) {
  ...blah...
}
```

like `stex.log` I want code to assume that it will be present and available.